src/lib/AnnotationControls.ts

```diff
@@ -59,6 +59,10 @@ export default class AnnotationControls {
         fullscreen.removeListener('exit', this.handleFullscreenExit);
     }
 
+    public focus(): void {
+        this.controlsElement.querySelector(`.${CLASS_REGION_BUTTON}`).focus();
```

Comment:
What happens if we have more than one button? Or if this button isn't available for a given viewer?